Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAttackSurfaceReductionRulesPolicyWindows10ConfigManager/MSFT_IntuneAttackSurfaceReductionRulesPolicyWindows10ConfigManager.psm1

```diff
@@ -8,7 +8,7 @@ function Get-TargetResource
         [System.String]
         $Identity,
```

Comment:
This should be required as well to match the *.schema.mof file of the resource.

Reply:
I have made the id as key because its the unique identifier of the resource and  display name as required for get-targetresource: it will first check for an id and then revert back to display name to find the resource
Displayname shouldn't be a key imo as it doesn't have to be unique in the Intune portal

Reply:
What’s there first: id or display name? If it is possible to create the element with the id, we could remove the need for a display name